x/bank/types/keys.go

```diff
@@ -33,7 +33,12 @@ var (
 	// ParamsKey is the prefix for x/bank parameters
 	ParamsKey = collections.NewPrefix(5)
 
+	// SupplyOffKey is a Osmosis specific key that handles supply offsets
 	SupplyOffsetKey = collections.NewPrefix(88)
+
+	// SupplyOffKeyOld is a Osmosis specific key that handles supply offsets pre v0.50
+	// TODO: Remove in the v28 upgrade
+	SupplyOffsetKeyOld = []byte{0x88}
```

Comment:
I just want to verify that you checked NewPrefix 88 does not collide with []byte{0x88}

Reply:
Yeah I did, this is the bug

Reply:
Ah, just understood, thanks